autogpts/evo.ninja/setup

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Ensure all commands are run within this script's directory
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd "$SCRIPT_DIR"
+
+# Clone the repo and checkout the release branch
+REPO="https://github.com/polywrap/evo.ninja"
+BRANCH="release/autogpt"
+
+echo "Cloning evo.ninja..."
+git init
+git remote add origin $REPO
+git pull
+git checkout $BRANCH
```

Comment:
I think this is equivalent to `git clone -b "$BRANCH" "$REPO"`

Reply:
No git clone assumes an empty directory, so this will fail to run because the `setup` file exists.